test/framework/functional/output_forward.go

```diff
@@ -95,7 +93,7 @@ func (f *CollectorFunctionalFramework) addForwardOutputWithConf(b *runtime.PodBu
 	}
 
 	log.V(2).Info("Adding container", "name", name)
-	b.AddContainer(name, utils.GetComponentImage(constants.FluentdName)).
+	b.AddContainer(name, "quay.io/openshift-logging/fluentd:5.9").
```

Comment:
nit, but still used in a few places, so why not add to constants package?   Also, did you remove the `FluentdName` and `FluentdImageEnvVar` from constants file?

Reply:
Updated where I could